lerobot/configs/env/aloha.yaml

```diff
@@ -3,7 +3,7 @@
 fps: 50
 
 env:
-  name: aloha
+  name: gym_aloha
```

Comment:
So why these changes?

Reply:
Main reason is removing this line:
https://github.com/huggingface/lerobot/pull/290/files/e7d4901301a35e18dcfcf6e4f04b68f84d2a929a#diff-0f7c1a6977da530f85035451866c98b1938bee512d68bfd4febfb1a8e367114eL30

Previously, there was an assumption that all package names of the environments were prefixed by `gym_`. In rlbench, this is not the case. I could either keep this assumption and have a special case for rlbench, or remove the assumption. I thought the latter was better.